ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,273 @@
+# Support for the YDB dialect in the Liquibase migration tool. 
+
+## Introduction 
+
+In today's world of software development, database schema versioning is a critical task to ensure consistency and the ability to roll back changes. Database migration tools, such as Liquibase, provide tools for versioning, tracking, and applying changes to the schema. 
```

Comment:
Given the term "consistency" is used so extensively in the context of consistency models and anomalies (like in https://jepsen.io/consistency), I'd recommend to rephrase this paragraph to avoid using it in the other meaning.